src/Core/Entities/PlayerEntityControls.m

```diff
@@ -991,7 +991,7 @@ - (void) pollApplicationControls
 		// snapshot
 		const BOOL *joyButtonState = [[OOJoystickManager sharedStickHandler] getAllButtonStates];
 		if (([self checkKeyPress:n_key_snapshot] || joyButtonState[BUTTON_SNAPSHOT]) &&
-			![gameView allowingStringInput] && // not while entering text on the keyboard config screens
+			(![gameView allowingStringInput] || gui_screen == GUI_SCREEN_LONG_RANGE_CHART) && // not while entering text on the keyboard config screens
```

Comment:
Proposal: Ensure that allowingStringInput is either alphanumeric or nothing, if so then take snapshot. This has the advantage of working also with future screens that may require text input and does not touch keyboard config or load/save screens.
[gameView allowingStringInput] <= gvStringInputAlpha

Reply:
Nice! Much better!